File: Leetcode/3477 - Fruits Into Baskets II/solution.js
var numOfUnplacedFruits = function(fruits, baskets) {
    let count = 0;
    for (let i = 0; i < fruits.length; i++) {
        let placed = false;
        for (let j = 0; j < baskets.length; j++) {
            if (baskets[j] >= fruits[i]) {
                baskets[j] = 0;
                placed = true;
                break;
            }
        }
        if (!placed) count++;
    }
    return count;
};

File: Leetcode/3477 - Fruits Into Baskets II/solution.py
def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
    count: int = 0
    for fruit in fruits:
        placed: bool = False
        for i in range(len(baskets)):
            if baskets[i] >= fruit:
                baskets[i] = 0
                placed = True
                break
        if not placed:
            count += 1
    return count